person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  # attribute accessor for getters and setters
+  attr_accessor :id, :name, :age
+
+  # constructor
+  def initialize(age, name = 'unknown', parent_permission: true) # rubocop:disable Lint/UnusedMethodArgument
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+  # private method
+
+  private
+
+  def of_age?
+    return true if age >= 18
+  end
+
+  # public method
+  def can_use_services?
+    return unless age || parent_permission
```

Comment:
- [ ] @Ibnballo1 great work in your class however you called the wrong class in your private method you created `of_age?` method but here you are calling the `age` method, also when you implement suggested changes in initialize method the parent permission will be passed as `@parent_permission`, same should be used for the age in your private method.

Reply:
I did not really understand this comment, please @danmainah